### ISSUE TITLE
[a11y] Audit Table

### ISSUE THREAD

**Issue by MartinWeb** [AxaFrance/react-toolkit#1144]
# Vérification de l'accessibilité du composant : Table

Afin d'évaluer le niveau d'accessibilité d'un composant, il faut effectuer les actions ci-dessous.

## Analyse statique d'accessibilité

Remplir les champs ci-dessous, en se basant sur l'onglet accessibilité de chaque composant dans le storybook.

**Table :**
![image](https://github.com/AxaFrance/react-toolkit/assets/1991349/e5e48416-70f4-43eb-922b-83a1e0ac4efa)

**VIOLATIONS** : 0
*INCOMPLETE* : 0
PASSED : 6

**Items :**
![image](https://github.com/AxaFrance/react-toolkit/assets/1991349/6e19b327-448e-4aa6-a298-4d211ab1a9cd)

**VIOLATIONS** : 0
*INCOMPLETE* : 0
PASSED : 7

**Pager (default) :**
![image](https://github.com/AxaFrance/react-toolkit/assets/1991349/5f7bd310-9011-4aa9-b6ac-a42a5f9d7bb6)

**VIOLATIONS** : 0
*INCOMPLETE* : 0
PASSED : 11

**Pager (light) :**
![image](https://github.com/AxaFrance/react-toolkit/assets/1991349/ce892e99-5039-43c0-b492-d71311becaea)

**VIOLATIONS** : 1 : pas de texte discernable pour les deux boutons précédent et suivant
*INCOMPLETE* : 0
PASSED : 10

En plus mettre une capture de l'onglet en commentaire de cette issue.

## Analyse manuelle

- [x] Est ce que le composant est navigable au clavier (tab) ?
- [x] Est ce que le composant est navigable avec NVDA ?
Ok, mais quelques choses à corriger : 
- Pour le pager, pas d'indication globale de la liste (peut être spécifier qu'il s'agit de la liste des pages disponibles pour le tableau).
- Pour le pager, NVDA lit les < > des boutons précédent et suivant.
- [ ] Est ce que le composant est navigable avec un autre lecteur d'écran ?

## Vérification sémantique

- [x] #1181
Il manque la possibilité de renseigner un `caption`
- [x] Vérifier la validité HTML W3C (https://validator.w3.org/#validate_by_input) ?
- [x] Vérifier la validité CSS (https://jigsaw.w3.org/css-validator/#validate_by_input) ?

## Problèmes détectés

Si vous avez détectés un défaut d'accessibilité, veuillez créer une issue décrivant le défaut et la rattacher à cette issue.

**Comment by MartinWeb** [comment]
Remontées à corriger :

- Pager light : pas de texte discernable pour les boutons précédent et suivant
- Lecteur d'écran NVDA : 
1. Pour le pager, pas d'indication globale de la liste (peut être spécifier qu'il s'agit de la liste des pages disponibles pour le tableau).
2. Pour le pager, NVDA lit les < > des boutons précédent et suivant.

**Comment by MartinWeb** [comment]
Report sur le projet design system
